app/controllers/planning_application/permitted_development_rights_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class PlanningApplication
-  class PermittedDevelopmentRightsController < AuthenticationController
+  class PermittedDevelopmentRightsController < AuthenticationController # rubocop:disable Metrics/ClassLength
```

Comment:
I’m wondering whether is worth of moving the `immunity  details` on their own controller

Reply:
I've created a new controller in my next PR as I'll need another show view so it makes sense 👍